deps/blas.mk

```diff
@@ -46,7 +46,7 @@ $(BUILDDIR)/$(OPENBLAS_SRC_DIR)/build-compiled: | $(BUILDDIR)/objconv/build-comp
 endif
 endif
 
-OPENBLAS_FFLAGS :=
+OPENBLAS_FFLAGS := $(FFLAGS) $(JFFLAGS)
 OPENBLAS_CFLAGS :=
```

Comment:
> Yes, but thats just recovering the status pre #20806

Not with CFLAGS - looking at https://github.com/JuliaLang/julia/blob/80b2da8137c71d8d53ce70a6b439fbc06fc05192~/deps/blas.mk CFLAGS is only ever appended to, never set to empty.